scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/DBTaskDataParameters.java

```diff
@@ -23,10 +24,12 @@
     private final boolean pending;
     private final boolean running;
     private final boolean finished;
+    private final SortSpecifierContainer sortParams;
+
     private final Set<TaskStatus> status;
 
     DBTaskDataParameters(String tag, long from, long to, int offset, int limit, String user,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)